lua/entities/gmod_wire_expression2/core/custom/sv_vex_main.lua

```diff
@@ -7,53 +7,32 @@
  Random Misc. Functions that are cool like hiding other people's chat (probably doesn't work) and setting the ranger Filter.                    
 ]]
 
-local E2Table = function() return {n={},ntypes={},s={},stypes={},size=0} end
-
-local function luaTablToE2(T)
```

Comment:
Refactored, moved into helpers, [click here to see](https://github.com/Vurv78/VExtensions/pull/15/files#diff-13513553927461b66e8fd31dbdf4979c1444325be4f59759e4217ca11732bd96R41).